ebib-utils.el

```diff
@@ -279,22 +279,31 @@ Ebib-specific mode line altogether."
   "Columns to display in the index buffer.
 Each column consists of the BibTeX field to be displayed, which
 is also the column's label, the column's maximum width and a flag
-indicating whether sorting on this column is possible.
+indicating whether and how sorting on this column is possible.
 
 Any BibTeX or biblatex field can be used as a label.  There are
 also two special labels: \"Entry Key\" and \"Author/Editor\".
 The label \"Entry Key\" displays the entry's BibTeX key, and the
 label \"Author/Editor\" displays the contents of the Author
 field, or, if that is empty, the contents of the Editor field.
 
+If the sorting flag is `t', then comparison for sorting is done
+with `string-collate-lessp'. If it is nil, sorting is impossible.
```

Comment:
In fact, with the latest commit, `t` means "look up the sorting function in `ebib-field-sort-functions-alist`, defaulting to `string-collate-lessp`.

Reply:
Fixed.